compiler-core/src/parse/snapshots/gleam_core__parse__tests__invalid_punning.snap

```diff
@@ -0,0 +1,12 @@
+---
+source: compiler-core/src/parse/tests.rs
+expression: "\npub fn main() {\n  wibble(:)\n}\n"
+---
+error: Syntax error
+  ┌─ /src/parse/error.gleam:3:10
+  │
+3 │   wibble(:)
+  │          ^ There must be a 'let' to bind variable to value
```

Comment:
This seems like an odd error

Reply:
Yeah it's really strange, it was not introduced by this PR and is already what the compiler is producing today. Do you think I could open an issue and address it in a different PR?

Reply:
Sounds good to me